tests/run-make/remap-path-prefix/rmake.rs

```diff
@@ -5,7 +5,7 @@
 
 // FIXME(Oneirical): check if works without ignore-windows
 
-use run_make_support::{invalid_utf8_contains, invalid_utf8_not_contains, is_darwin, rustc};
+use run_make_support::{fs_wrapper, is_darwin, rustc};
```

Comment:
I would still rename `invalid_utf8_contains` to e.g. `contains_bytes`, but doesn't need to be in this PR.